graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -67,22 +73,26 @@
 
 
 def rename_schema(
```

Comment:
For fields, I'll probably add to either an existing visitor or create a new one and call a function within here, rather than have two separate functions. I suppose we could then call it something more general like `manipulate_schema` but it might not be clear that we're allowing both suppression and renamings here.